opsdroid/cli/config.py

```diff
@@ -58,35 +58,64 @@ def edit(ctx):
 
 @config.command()
 @click.pass_context
-@click.argument("path", required=False, type=click.Path(exists=True))
+@click.option(
+    "-f",
+    "path",
+    help="Validates opsdroid configuration from a path.",
+    type=click.Path(exists=True),
+)
```

Comment:
You've got a lot of code duplication here. I wonder if we can just set this option on the root level click group. It would make sense to have the `-f` flag is most places.